causal_testing/testing/causal_test_result.py

```diff
@@ -43,18 +43,27 @@ def __init__(
             self.effect_modifier_configuration = {}
 
     def __str__(self):
+        def push(s, inc="  "):
+            return inc + str(s).replace("\n", "\n" + inc)
+
+        result_str = " " + str(self.test_value.value)
+        if "\n" in result_str:
+            result_str = "\n" + push(self.test_value.value)
         base_str = (
             f"Causal Test Result\n==============\n"
             f"Treatment: {self.estimator.treatment}\n"
             f"Control value: {self.estimator.control_value}\n"
             f"Treatment value: {self.estimator.treatment_value}\n"
             f"Outcome: {self.estimator.outcome}\n"
             f"Adjustment set: {self.adjustment_set}\n"
-            f"{self.test_value.type}: {self.test_value.value}\n"
+            f"{self.test_value.type}:{result_str}\n"
```

Comment:
Very minor, would you want a space after the colon?

Reply:
Yes, good spot